app.rb

```diff
@@ -0,0 +1,106 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'rental'
+# require_relative 'classroom'
```

Comment:
- [ ] Kindly make sure to remove all the commented code from your code base so it looks clean and easy to navigate, as for now lines `5` and `7` are commented